### PR TITLE
Port release/gefs_v12 to WCOSS2

### PR DESCRIPTION
**Pull request by XianwuXue-NOAA**

### CODE REVIEW

File: sorc/checkout.sh [hunk 1]
```diff
@@ -26,11 +26,11 @@ err=0
 echo fv3gfs checkout ...
 if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
-    git clone --recursive gerrit:EMC_FV3-GSDCHEM-WW3 fv3gfs.fd >> ${LOG_DIR}/checkout-fv3gfs.log 2>&1
+    git clone https://github.com/NOAA-EMC/FV3-GSDCHEM-WW3.git fv3gfs.fd >> ${LOG_DIR}/checkout-fv3gfs.log 2>&1
     rc=$?
     ((err+=$rc))
     cd fv3gfs.fd
-	git checkout gefs_v12.1.0
+	git checkout release/gefs_v12
```
Comment (WalterKolczynski-NOAA) [inline sorc/checkout.sh]: This should be the new tag

File: sorc/enkf_chgres_recenter_nc.fd/src/makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 SHELL=  /bin/sh
 
-LIBS= $(FV3GFS_NCIO_LIB) $(BACIO_LIB4) $(W3NCO_LIB4) $(IP_LIB4) $(SP_LIB4) -L$(NETCDF)/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5 -lz
+LIBS= $(FV3GFS_NCIO_LIB) $(BACIO_LIB4) $(W3NCO_LIB4) $(IP_LIB4) $(SP_LIB4) -L$(NETCDF)/lib -lnetcdff -lnetcdf -lz # -lhdf5_hl -lhdf5
```
Comment (WalterKolczynski-NOAA) [inline sorc/enkf_chgres_recenter_nc.fd/src/makefile]: Remove the commented libs

File: sorc/link_fv3gfs.sh [hunk 3]
```diff
@@ -38,6 +38,8 @@ elif [ $machine = "theia" ]; then
     FIX_DIR="/scratch4/NCEPDEV/global/save/glopara/git/fv3gfs/fix"
 elif [ $machine = "hera" ]; then
     FIX_DIR="/scratch1/NCEPDEV/global/glopara/fix"
+elif [ $machine = "wcoss2" ]; then
+    FIX_DIR="/lfs/h2/emc/global/noscrub/Kate.Friedman/glopara/FIX/fix_nco_gfsv15"
```
Comment (WalterKolczynski-NOAA) [inline sorc/link_fv3gfs.sh:42]: @KateFriedman-NOAA is this the final resting place for GFS v15 fix data?

File: sorc/link_fv3gfs.sh [hunk 4]
```diff
@@ -180,12 +183,12 @@ if [ $machine = dell -o $machine = hera ]; then
     done
 fi
 
-for ufs_utilsexe in chgres_cube nemsio_get nemsio_read global_chgres ; do
+for ufs_utilsexe in chgres_cube ; do # nemsio_get nemsio_read global_chgres ; do
```
Comment (WalterKolczynski-NOAA) [inline sorc/link_fv3gfs.sh]: Remove old executables

File: sorc/link_fv3gfs.sh [hunk 5]
```diff
@@ -201,7 +204,7 @@ $LINK ../sorc/gsd_prep_chem.fd/workflow/emc-global/exec/prep_chem_sources_RADM_F
 
 cd ${pwd}/../sorc   ||   exit 8
     $SLINK gsi.fd/util/EnKF/gfs/src/calc_increment_ens.fd                                  calc_increment_ens.fd
-    $SLINK gsi.fd/util/EnKF/gfs/src/calc_increment_ens_gsdchem.fd                          calc_increment_ens_gsdchem.fd
+    #$SLINK gsi.fd/util/EnKF/gfs/src/calc_increment_ens_gsdchem.fd                          calc_increment_ens_gsdchem.fd
```
Comment (WalterKolczynski-NOAA) [inline sorc/link_fv3gfs.sh]: Remove

File: sorc/link_fv3gfs.sh [hunk 6]
```diff
@@ -217,13 +220,13 @@ cd ${pwd}/../sorc   ||   exit 8
 
     $SLINK gfs_post.fd/sorc/ncep_post.fd                                                   gfs_ncep_post.fd
 
-    $SLINK ufs_utils.fd/sorc/fre-nctools.fd/tools/shave.fd                                 shave.fd
-    for prog in filter_topo fregrid make_hgrid make_solo_mosaic ; do
+    #$SLINK ufs_utils.fd/sorc/fre-nctools.fd/tools/shave.fd                                 shave.fd
```
Comment (WalterKolczynski-NOAA) [inline sorc/link_fv3gfs.sh]: Remove

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: sorc/link_fv3gfs.sh (hunk 3 above)
Comment (KateFriedman-NOAA) [reply sorc/link_fv3gfs.sh:42]: @WalterKolczynski-NOAA No, I need the global group account space to host these. That will be the final resting place, my copies are meant to be temporary. @arunchawla-NOAA Any update on setting up the global group account on WCOSS2? We have a bunch of stuff to host under it, like fix files for hand-off.

File: sorc/checkout.sh (hunk 1 above)
Comment (XianwuXue-NOAA) [reply sorc/checkout.sh]: Yes, jut updated.

File: sorc/enkf_chgres_recenter_nc.fd/src/makefile (hunk 2 above)
Comment (XianwuXue-NOAA) [reply sorc/enkf_chgres_recenter_nc.fd/src/makefile]: No problem. I will delete them in the next commit

File: sorc/link_fv3gfs.sh (hunk 4 above)
Comment (XianwuXue-NOAA) [reply sorc/link_fv3gfs.sh]: No problem. I will delete in the next commit.

File: sorc/link_fv3gfs.sh (hunk 5 above)
Comment (XianwuXue-NOAA) [reply sorc/link_fv3gfs.sh]: done in the next commit

File: sorc/link_fv3gfs.sh (hunk 6 above)
Comment (XianwuXue-NOAA) [reply sorc/link_fv3gfs.sh]: same as before.

File: sorc/link_fv3gfs.sh (hunk 3 above)
Comment (XianwuXue-NOAA) [reply sorc/link_fv3gfs.sh:42]: Thanks @WalterKolczynski-NOAA and @KateFriedman-NOAA I am also waiting for the final resting place to freeze the code GEFSv12

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Good to merge. No tag until fix gets a final location.

**Comment by XianwuXue-NOAA** [comment]
> Good to merge. No tag until fix gets a final location.

Agree. Once I got the final resting location of the fix folder, I will commit it.

**Pull request merged**